### ISSUE TITLE
Mouse wheel action is laggy and unresponsive on chart

### ISSUE THREAD

**Issue by dez1729** [amcharts/amcharts5#204]
Using example
https://codepen.io/team/amcharts/pen/eYGvOEQ

When trying to zoom with the mouse wheel, it can become unresponsive and laggy.

**Comment by martynasma** [comment]
It seems that adding custom `globalpointermove` event gets in the way of reliably firing it used by internal hover check mechanism, hence wheel working unpredictably.

We will work on a fix.

That said, I think for your specific purpose you'd be better off attaching generic hover/blur events on the chart div to toggle scrollbar. E.g.:

```JavaScript
root.dom.addEventListener("pointerover", function() {
  scrollbar.show();
});

root.dom.addEventListener("pointerout", function() {
  scrollbar.hide();
});
```

**Comment by dez1729** [comment]
I think the issue persists when I replaced the globalpointermove with the pointerover and pointerout events.  Also, seems like the pointerover/pointerout events are giving inconsistent results (fades out right away / doesn't always fade in).

**Comment by martynasma** [comment]
Did you remove the `globalpointermove` event handler altogether?

I can't reproduce the lag when I replace it with the code I suggested earlier.

**Comment by dez1729** [comment]
I reproduced it in the example above by removing the 'globalpointermove' event and adding the 'pointerover' and 'pointerout' events.

**Comment by martynasma** [comment]
I can't seem to be able to reproduce it in Chrome. Any chance are you using something else?

**Comment by dez1729** [comment]
Tested on chrome with two different machines.

**Comment by martynasma** [comment]
Nobody here can't reproduce it without the event. Can you post new CodePen? Perhaps we're doing something differently.

**Comment by dez1729** [comment]
OK I think I made a mistake and I used chart.events.on instead of root.dom.addEventListener, as you did above. After the change it works fine.

**Comment by martynasma** [comment]
> OK I think I made a mistake and I used chart.events.on instead of root.dom.addEventListener, as you did above. After the change it works fine.

Phew, OK :)

We'd still like to fix this so that adding events would not interfere with usability of the chart. Reopening.

**Comment by martynasma** [comment]
Fixed in [5.1.8](https://github.com/amcharts/amcharts5/releases/tag/5.1.8).

## [5.1.8] - 2022-03-10

### Added
- Settings `pinchZoomX` and `pinchZoomY` (defaults: `false`) added to `XYChart`. If set to `true` will allow pinch-zooming of a chart on touch devices. NOTE: these settings are not supported on `RadarChart`.
- `topPlotContainer` added to `XYChart`. It's a `Container` that can be used to place element over plot area.

### Changed
- The types for adapters has been changed to be more intuitive and useful. ([Issue 312](https://github.com/amcharts/amcharts5/issues/312))
- Tapping an element on a touch device will now generate `pointerover` event in addition to `click`. When tapped somewhere else, `pointerout` event will be triggered.

### Fixed
- Fixing ES6 module detection by BundlePhobia again ([Issue 294](https://github.com/amcharts/amcharts5/issues/294)).
- Pinch-zooming of a `MapChart` was not working well in some cases.
- `maxHeight` setting in `Sprite` was not working.
- Change of `minWidth` was not being applied at once, only if some other changes triggered chart to be redrawn.
- Changing or not setting data on an existing `FlowChart` was resulting in a JS error.
- `maxPanOut` setting in `MapChart` was not working.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
